test/lib/Comparator.h

```diff
@@ -0,0 +1,73 @@
+#pragma once
+#include <libstuff/libstuff.h>
```

Comment:
Let's not make tpunit++ dependent on libstuff, just include the specific headers we'll need here.